lib/datadog/core/configuration/components.rb

```diff
@@ -345,6 +347,23 @@ def should_enable_gc_profiling?(settings)
               false
             end
           end
+
+          def print_new_profiler_warnings
+            if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.6')
+              Datadog.logger.warn(
+                'New Ruby profiler has been force-enabled. This feature is in alpha state. Please report any issues ' \
+                'you run into!'
+              )
+            else
+              # For more details on the issue, see the "BIG Issue" comment on `gvl_owner` function in
+              # `private_vm_api_access.c`.
+              Datadog.logger.warn(
+                'New Ruby profiler has been force-enabled on a legacy Ruby version (< 2.6). This is not recommended in ' \
+                'production environments, as due to limitations in Ruby APIs, we suspect it may lead to crashes in very ' \
+                'rare situations. Please report any issues you run into!'
```

Comment:
Should the message include a conveninence link to `https://github.com/DataDog/dd-trace-rb/issues/new`?